delta/plugins/archive/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/archive/routes/ArchiveRoutes.scala

```diff
@@ -96,9 +97,9 @@ class ArchiveRoutes(
       }
     }
 
-  private def sourceToFileResponse(source: AkkaSource, format: ArchiveFormat[_]): FileResponse =
-    FileResponse(s"archive.${format.fileExtension}", format.contentType, 0L, source)
+  private def sourceToFileResponse(source: AkkaSource): FileResponse =
+    FileResponse(s"archive.${Zip.fileExtension}", Zip.contentType, 0L, source)
 
-  private def archiveResponse: Directive1[Option[ArchiveFormat[_]]] =
-    extractRequest.map(ArchiveFormat(_))
+  private def archiveResponse: Directive1[Option[Zip.type]] =
+    extractRequest.map(Zip(_))
```

Comment:
I think this should be a boolean if we're not using the `Some` ?

Reply:
Ah yeah definitely